go.mod

```diff
@@ -3,53 +3,117 @@ module github.com/openpubkey/signed-attestation
 go 1.21.1
 
 require (
+	github.com/go-openapi/strfmt v0.21.8
 	github.com/in-toto/in-toto-golang v0.9.0
 	github.com/lestrrat-go/jwx/v2 v2.0.18
-	github.com/openpubkey/openpubkey v0.2.2-0.20231211101749-b939cc536097
+	github.com/openpubkey/openpubkey v0.2.2-0.20231213094744-f54cdc5f6dc9
```

Comment:
Should we cut a release?

Reply:
I would recommend it, there are some important fixes between those versions. cc: @lgmugnier